db/migrate/20170901163452_add_availability_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvailabilityToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_column :users, :availability, :boolean
```

Comment:
Hm, personal preference, but I would suggest to **always** put a `null: false` and a default value (`false` in this case) to a boolean value. You'll otherwise end up with a 3 state boolean: `null, true, false` - which can be a potential source of problems later 😉